sktime/datasets/_fpp3_loaders.py

```diff
@@ -349,6 +362,8 @@ def load_fpp3(dataset, temp_folder=None):
     temp_folder: str, optional
         Location of temporary data folder for downloading and extracting the dataset.
         Deleted if the operation is successful.
+    robust: logical, optional (default=True)
```

Comment:
if you need this parameter, I would suggest to split the function into a public and a private function, the internal parameter is only there in the private function.

That way, we do not add a public parameter that we tell users to ignore.

Let me know if I am not very clear in explaining.